pypeit/calibrations.py

```diff
@@ -1190,16 +1291,27 @@ def get_tilts(self):
         self.wavetilts.to_file()
         return self.wavetilts
 
-    def run_the_steps(self):
+    #def run_one_step(self, step, force:str=None):
+    #    getattr(self, f'get_{step}')(force=force)
+
+    def run_the_steps(self, stop_at_step:str=None):
         """
         Run full the full recipe of calibration steps.
         """
         self.success = True
         for step in self.steps:
-            getattr(self, f'get_{step}')()
+            #self.run_one_step(step)
+            if stop_at_step is not None and step == stop_at_step:
+                force = 'remake'
+            else:
+                force = None
```

Comment:
It doesn't seem `force=reload` is actually an option. Should we remove it from the if loop in each step?